thoth/storages/graph/postgres.py

```diff
@@ -4429,6 +4429,7 @@ def sync_security_indicator_aggregated_result(self, document: dict) -> None:
                 session,
                 si_aggregated_run_document_id=document_id,
                 datetime=metadata["datetime"],
+                error=document.get("error", False),
```

Comment:
isn't it inside `metadata`? https://github.com/thoth-station/si-aggregator/pull/35/files#

Reply:
@KPostOffice please also adjust queries that retrieve si analyzed results used in adviser and retrieve packages not si analyzed for graph-refresh

Reply:
> isn't it inside `metadata`? https://github.com/thoth-station/si-aggregator/pull/35/files#

No, it is in the regular output. We just gen metadata on error since it is reliant on a successful bandit run.